app/src/main/java/com/example/mystore/ui/shared/Divider.kt

```diff
@@ -0,0 +1,29 @@
+package com.example.mystore.ui.shared
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Divider
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.unit.Dp
+import com.example.mystore.R
+import com.example.mystore.ui.theme.LightGray40
+
+@Composable
+fun CustomDivider(
+    modifier: Modifier = Modifier
```

Comment:
You're exposing a modifier here which is nice. But you're attaching values only to the default value. If you were to submit a modifier from outside, you'd wipe all of them out. Just `modifier: Modifier = Modifier` is sufficient.

Reply:
What if the logic is to have a default values and if there is a need to ovveride them from the outside, or is it a bad practice?

Reply:
It's generally best to have modifiers that will never change no matter the circumstances internally and then expose a modifier that you combine with the internal one for things on top like outer paddings, etc. Having overridable default values makes sense for non-modifier arguments but not for modifiers.